core/modules/config/config.admin.inc

```diff
@@ -542,6 +542,11 @@ function config_get_modified_times() {
  *
  * @return array
  *   An array of formatted strings showing the diffs between the two storages.
+ * 
+ * @see config_set()
+ * @see config_get()
+ * @see config_clear()
+ * @see config_menu()
```

Comment:
I'm not sure these `@see` values are directly related to this. Perhaps a `@see config_diff_page()` instead of all of these?

Reply:
Done